src/main/resources/expat-site-default.xml

```diff
@@ -85,6 +95,12 @@
         </client>
     </hops>
 
+    <epipe>
+        <path></path>
+        <!-- indices: projects/featurestore/app_provenance -->
+        <reindex></reindex>
+    </epipe>
```

Comment:
how will the reindexing work with 
https://github.com/logicalclocks/elasticsearch-chef/blob/master/attributes/default.rb#L61
probably, you would need to move the logic out chef to avoid repetition. Also, another point do we need/have to reindex the projects/featurestore on every upgrade? or only do so when there are changes to schema?
https://github.com/logicalclocks/epipe-chef/blob/master/templates/default/config-reindex.ini.erb#L3
https://github.com/logicalclocks/epipe-chef/blob/master/recipes/default.rb#L69